ideascube/monitoring/templates/monitoring/specimen_confirm_delete.html

```diff
@@ -1,4 +1,4 @@
 {% extends 'ideascube/form_confirm_delete.html' %}
 {% load i18n %}
 
-{% block confirm_message %}{% blocktrans with specimen=specimen.barcode item=specimen.item %}Are you sure you want to delete the specimen "{{ specimen }}" of "{{ item }}"?{% endblocktrans %}{% endblock %}
+{% block confirm_message %}{% blocktrans with specimen=specimen.pretty_identifier item=specimen.item %}Are you sure you want to delete the specimen "{{ specimen }}" of "{{ item }}"?{% endblocktrans %}{% endblock %}
```

Comment:
I really think it would be better to display both the `barcode` and the `serial` here, just like in #659.